egov-indexer/egov-vendor.yaml

```
@@ -17,7 +17,7 @@ ServiceMaps:
         - inJsonPath: $
           outJsonPath: $.Data.vendor
         mdmsMapping:
-        - path: http://egov-mdms-service.pucar-prod:8080/egov-mdms-service/v1/_search
+        - path: http://egov-mdms-service.egov:8080/egov-mdms-service/v1/_search
```

yamllint (1.35.1)

[error] wrong indentation: expected 9 but found 8

(indentation)